city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    def parse(self, response):
+        """   `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping needs.   """
+        meeting_location = ''
+
+        for item in response.css("#content-232764 div.panel-body p"):
+
+            if item.css("p > strong ::text").getall():
+                meeting_location = self._parse_location(item)
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=meeting_location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+            meeting['status'] = self._get_status(meeting)
+            meeting['id'] = self._get_id(meeting)
+            print(str(meeting))
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        item_txt = ''.join(item.css("p::text").getall())
```

Comment:
Since you're pulling `item_txt` in separate functions, it might make sense to create this variable in the loop and then pass it to these functions separately

Reply:
Except that I'm using different parts in each function.